lib-utilities/config/config.go

```diff
@@ -110,6 +110,7 @@ type APIGatewayConf struct {
 // AddComputeSkipResources stores all resource which need to igonered while adding Computer System
 type AddComputeSkipResources struct {
 	SystemCollection  []string `json:"SystemCollection"`  // holds the value of  system resource which need to be ignored
+	ManagerCollection []string `json:"ManagerCollection"` // holds the value of  manager resource which need to be ignored
```

Comment:
why ignore these ? I think all these are badly named. Shouldnt they reflect the intended use ?

Reply:
The collections are used for ignoring the links links from the list of all the links present for that URI

Reply:
Action the second comment to rename these

Reply:
Why was the comment marked as resolved when nothing was changed?

Reply:
Addressed comments - Modified the list names